.flake8

```
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 88
```

As per the .editorconfig it should be set to 180
```suggestion
max-line-length = 180
```

---

This may be reduced from 180 in the future, but for now settings should match the .editorconfig

---

Ah, didn't see you had an editor config. 180 might be the largest max_line_length I've seen in a project. ;)